### ISSUE TITLE
Bug in learning.py

### ISSUE THREAD

**Issue by theSparta** [aimacode/aima-python#285]
In the function `LinearLearner` in `learning.py`, `learning_rate` is not used in the gradient update rule.

**Comment by coderark** [comment]
Hi, @theSparta can I work on this bug.
Only, need to correct the update rule right?

**Comment by theSparta** [comment]
#284 resolves this.
